### PR TITLE
Update changelog and testing instructions with 3.1.0-rc.1 release

### PR DESCRIPTION
**Pull request by ilyasfoo**
Update changelog and testing instructions with changes of 3.1.0-rc.1 release.

No changelog necessary.

### CODE REVIEW

File: TESTING-INSTRUCTIONS.md [hunk 1]
```diff
@@ -23,15 +23,47 @@
...
+7. Observe **WooCommerce Payments** is displayed in the suggested extensions
+8. Without refreshing, go back to the **Industry** step
+9. Select **CBD industry** and click on continue until Business Details step again
+10. In the extension list, observe that WooCommerce Payments is **NOT** displayed.
+
+**Change store country**
+
+11. Repeat steps 3~7
+12. Without refreshing, go back to the **Store Details** step
+13. Choose any store location that **doesn't** supports WCPay, such as United Kingdom (UK)
```
Comment (chihsuan) [inline TESTING-INSTRUCTIONS.md:56]: @ilyasfoo Could you help me replace the “United Kingdom” with other countries? Thanks. 🙏

**Review by chihsuan — APPROVED** [review]
Thanks @ilyasfoo. Pre-approving. Only a minor thing need to be fixed. 👍

File: TESTING-INSTRUCTIONS.md (hunk 1 above)
Comment (ilyasfoo) [reply TESTING-INSTRUCTIONS.md:56]: Ah, I see what you mean. I'll replace it with Malaysia since I know WCPay doesn't support us yet

Comment (ilyasfoo) [reply TESTING-INSTRUCTIONS.md]: Fixed in 47d2d6b

**Pull request merged**